tests/integration/test_data/generic_maven_e2e/container/Containerfile

```diff
@@ -0,0 +1,5 @@
+FROM ibmjava:11-jdk
```

Comment:
Is there any specific reason to use a short-name? It could be my setup, but I got a test failure due to short-name resolution being enforced. The test passed for me when I changed this to `FROM docker.io/ibmjava:11-jdk`.

Reply:
No reason really, changed to a fully qualified name.